qiskit/visualization/qcstyle.py

```diff
@@ -22,12 +22,15 @@ class DefaultStyle:
 
     def __init__(self):
         # Set colors
-        basis_color = '#FA74A6'     # Red
-        clifford_color = '#6FA4FF'  # Blue
-        non_gate_color = '#000000'  # Black
-        other_color = '#BB8BFF'     # Purple
-        pauli_color = '#05BAB6'     # Green
-        iden_color = '#05BAB6'      # Green
+        basis_color = '#FA74A6'               # Red
+        clifford_color = '#6FA4FF'            # Blue
+        iden_color = '#05BAB6'                # Green
+
+        hadamard_color = '#DC143C'            # Red
+        classical_gate_color = '#000080'      # Dark Blue
+        phase_gate_color = '#1E90FF'          # Light Blue
+        non_unitary_gate_color = '#808080'    # Grey
+        other_color = '#99004C'               # Dark Red
```

Comment:
These are the official color codes, could you update these?
```
classical: #002D9C
phase: #33B1FF
hadamard: #FA4D56
quantum: #9F1853
non-unitary: #A8A8A8
```

Reply:
@cryoris absolutely